tests/codeception/acceptance/Content/NewsCest.php

```diff
@@ -360,6 +305,51 @@ public function testRelatedNewsPerson(AcceptanceTester $I) {
     $I->cantSee($otherNews->label());
   }
 
->>>>>>> 12.x
+  /**
```

Comment:
Thank you, @pookmish 

![Thank you](https://media.giphy.com/media/v1.Y2lkPTc5MGI3NjExM2JkMndmeTBwZmFiMXhxNnZqY2swamNtdTBxN3htbjBva3Rzbm1maiZlcD12MV9naWZzX3NlYXJjaCZjdD1n/i21tixUQEE7TEqwmYa/giphy.gif)